aea/aea_builder.py

```diff
@@ -1338,6 +1339,8 @@ def from_aea_project(
         )
         builder = AEABuilder(with_default_packages=False)
 
+        load_env_file(str(aea_project_path / DEFAULT_ENV_DOTFILE))
```

Comment:
notice that we don't need to check the existence of the dotenv file. The library function `dotenv.load_dotenv` will handle that.